utils.go

```diff
@@ -40,3 +40,35 @@ func validateMountOptions(mountOptions []string) error {
 	}
 	return nil
 }
+
+func applyNameOperation(oldNames []string, names []string, op updateNameOperation) ([]string, error) {
+	result := make([]string, 0)
+	switch op {
+	case setNames:
+		// ignore all old names and just return new names
+		return names, nil
+	case removeNames:
+		// remove given names from old names
+		for _, name := range oldNames {
+			// only keep names in final result which do not intersect with input names
+			// basically `result = oldNames - names`
+			nameExistsInOldNames := false
```

Comment:
`nameExistsInOldNames` is guaranteed true by the loop containing this line.

Isn’t this variable actually `nameShouldBeRemoved`, or `nameExistsInNames`?[1]


([1] an illustration for the rationale of naming the input value `opParameters`, to make it distinct from all the “names” variables around.)

Reply:
I also think `nameShouldBeRemoved` is better, and switched `names` -> `opParameters` for better readability.